prerequisites/build-functions/set_or_print_url.sh

```diff
@@ -10,6 +10,12 @@ set_or_print_url()
   # Get package name from argument passed with  -p, -D, -P, -V, or -U
   package_to_build="${arg_p:-${arg_D:-${arg_P:-${arg_U:-${arg_V:-${arg_B}}}}}}"
 
+if [[ ! -z ${arg_u:-}  ]]; then
```

Comment:
I think `[[ -n "${arg_u:-}" ]]` would be clearer.... `-n` evaluates true when the string/var is non-null/non-empty